src/lib/vibrate.ts

```diff
@@ -0,0 +1,14 @@
+import { HapticLevel, mobileVibrate } from '@/context/WalletProvider/MobileWallet/mobileMessageHandlers'
+import { isMobile } from './globals'
+
+export const vibrate = (level: HapticLevel): Promise<void> | boolean => {
+	if (isMobile) {
+		return mobileVibrate(level)
+	}
+
+	if ('vibrate' in navigator) {
+		return navigator.vibrate(200)
+	}
+
+	return false
+}
```

Comment:
The only interesting logic, everything else is just adding vibrate in any handlers needed